docs/infrastructure/logging.md

```diff
@@ -135,6 +135,26 @@ Any custom log level is therefore defined with `-Dx.y.Z.Logger.level` where `x`,
 properties always have a higher priority over those defined in the
 `application.conf` file.
 
+### Truffle loggers
+
+[TruffleLogger](https://www.graalvm.org/truffle/javadoc/com/oracle/truffle/api/TruffleLogger.html)
+is designed to be independent of any other logging facilities. Setting a custom
+level for Truffle logger can be done programmatically via
+[ContextBuilder.option](<https://www.graalvm.org/truffle/javadoc/org/graalvm/polyglot/Context.Builder.html#option(java.lang.String,java.lang.String)>)
+polyglot context builder option, or via
+`polyglot.log.<language-id>.<class-name>.level` system property. Note that it is
+not enough to set just `-Dpolyglot.log.<language-id>.<class-name>.level` system
+property, one has to also specify this level via for forwarded log messages via
+`-D<language-id>.<class-name>.Logger.level` prop.
```

Comment:
The previous text is fine, but this last sentence completely confuses me. There are some non-English expressions like "via for forward" confuse me. Also I am missing explanation, why two properties are needed?